index.html

```diff
@@ -288,6 +288,8 @@ <h3>
           </li>
         </ol>
         <p data-link-for="VisibilityState">
+          The user agent MAY return "<a>hidden</a>" if the window is fully obscured 
+          by an accessibility tool but the accessibility tool is not covering it.
```

Comment:
@siusin what is the motivation for this? As in, does any browser actually do this? 

/cc @toddreifsteck

Reply:
See record of the TPAC discussion: https://www.w3.org/2017/11/07-webperf-minutes.html#item05

@toddreifsteck may know more details about this issue.

Reply:
This text is not quite accurate.

It should read something like.... 'if the window is fully obscured. It MUST NOT return "hidden" if the cause of cause of obscure is an accessibility tool.'

Edge has been evaluating and working on early code to do this.

Reply:
@toddreifsteck ah, interesting. "MUST NOT" is a strong statement, I'd love to hear from other implementers on whether this is implementable across all platforms (and OS's).

/cc @bzbarsky @rniwa @tdresser

Reply:
301 @dbolter

Reply:
@igrigorik I think you are missing my point because I'm not being clear. :-) 

Previously text was not flexible enough to allow innovation on various platforms. NOT marking "hidden" for a scenario due to platform limitations will hold us back when the web platform could improve.

Edge will be marking some documents to hidden in the near future when the monitor is off but the lock screen is not visible. That is a case that is not allowed by the current specification. It is my assertion that the specification's current language should be loosened to allow a UA to be more aggressive.

Previous text----
Return "visible" if: 
The user agent is not minimized and doc is the foreground tab. 
The user agent is fully obscured by an accessibility tool, like a magnifier, but a view of the doc is shown. 
Return "hidden" if: 
The user agent is minimized. 
The user agent is not minimized, but doc is on a background tab. 
The user agent is to unload doc. 
The Operating System lock screen is shown.

Reply:
@siusin Is it clear what we are asking to be updated in the PR based on the above text? Please let me know if there are additional questions.